recipes/cuda_tileiras/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -ex
+
+[[ -d lib64 ]] && mv lib64 lib
+
+[[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+# https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html?highlight=tegra#cross-compilation
+[[ ${target_platform} == "linux-aarch64" && ${arm_variant_type} == "sbsa" ]] && targetsDir="targets/sbsa-linux"
+[[ ${target_platform} == "linux-aarch64" && ${arm_variant_type} == "tegra" ]] && targetsDir="targets/aarch64-linux"
+
+if [ -z "${targetsDir+x}" ]; then
+    echo "target_platform: ${target_platform} is unknown! targetsDir must be defined!" >&2
+    exit 1
+fi
+
+mkdir -p ${PREFIX}/${targetsDir}
+
+#./tileiras --version command causes segfault, so do not apply patchelf
+#for i in `ls`; do
+#    [[ $i == "build_env_setup.sh" ]] && continue
+#    [[ $i == "conda_build.sh" ]] && continue
+#    [[ $i == "metadata_conda_debug.yaml" ]] && continue
+#
+#    if [[ $i == "bin" ]]; then
+#        for j in `ls "${i}"`; do
+#	          [[ -f "bin/${j}" ]] || continue
+#
+#            echo patchelf --force-rpath --set-rpath "\$ORIGIN/../lib:\$ORIGIN/../${targetsDir}/lib" "${i}/${j}" ...
+#            patchelf --force-rpath --set-rpath "\$ORIGIN/../lib:\$ORIGIN/../${targetsDir}/lib" "${i}/${j}"
+#        done
+#    fi
+#done
```

Comment:
> Looking at the ldd output it seems setting RPATH isn't needed anyhow, but I'd appreciate someone to weigh in on this

You are correct. We only need to use patchelf when installing binaries to `$PREFIX/$targetsDir/` because the targets directory is a non-standard conda install path, so if we don't modify the rpath in the binaries, they won't be able to find their dependencies. You know this already 😄.

Since this binary is installed directly into $PREFIX/bin, we also don't need most of this script including creating the targets directory, and we don't need patchelf as a dependency.